lib/ember_cli/runner.rb

```diff
@@ -32,9 +34,13 @@ def run!(command)
 
     private
 
-    def write(output, streams:)
-      streams.each do |stream|
-        stream.write(output)
+    def redirect(stream, pipes:)
+      pipes.each do |pipe|
+        Thread.new do
+          Thread.current.abort_on_exception = true
+
+          IO.copy_stream(stream, pipe)
+        end
```

Comment:
@jferris on top of that, now that we've introduce asynchrony, what's a good way to test this?